tests/run.sh

```diff
@@ -62,6 +62,13 @@ do
     echo "$(tput setaf 1)FAILED$(tput sgr 0)"
     FAILED_TESTS=$((FAILED_TESTS+1))
   fi
+
+  if [ -v LOG_FILE_PATH ]; then
+    set -e
```

Comment:
Do you want to fail the test run if the copy fails? Also, -e is already active here, so you are switching to +e

Reply:
will fix